omnigibson/systems/micro_particle_system.py

```diff
@@ -1132,6 +1242,9 @@ def _sync_particle_instancers(cls, idns, particle_groups, particle_counts):
             info = idn_to_info_mapping[idn]
             cls.generate_particle_instancer(idn=idn, particle_group=info["group"], n_particles=info["count"])
 
+        # Update the max particle instancer ID
+        cls.update_max_instancer_idn()
```

Comment:
This line should be put in `generate_particle_instancer`.

Reply:
done!